src/Highlighter.ts

```diff
@@ -11,6 +11,7 @@ interface IOptions {
   snapMathJax?: boolean;
   snapWords?: boolean;
   className?: string;
+  skipIDsBy?: RegExp[];
```

Comment:
not sure its necessary to make this a list of regexes, we can just combine our criteria into one regex